apps/expo/src/lang/en-US.json

```diff
@@ -234,52 +234,52 @@
       },
       "energy_query": {
         "information_screen": {
-          "title": "EnergyQuery Information",
-          "button": "Understood!"
+          "title": "Clarification",
+          "button": "Ok"
         },
         "homeselect_screen": {
           "title": "Select your Home",
-          "subtitle": "Drag the map around and place the pin above your home",
-          "button": "Use this location!"
+          "subtitle": "Drag and zoom the map to place the pin on your home",
+          "button": "Use this location"
         },
         "homeaddress_screen": {
           "title": "Fill in your address",
-          "subtitle": "Drag the map around and place the pin above your home.\nYou can also edit your address if you need to after.",
-          "button": "Use this address!",
+          "subtitle": "Drag and zoom the map to place the pin on your home or edit the address.",
+          "button": "Use this address",
           "error": {
             "title": "Incorrect address",
-            "message": "This is not an address we can ues.\nPlease try again and check the manual and FAQ for more info."
+            "message": "This does not seem a valid address.\nPlease try again. Check the manual or FAQ for more info."
           }
         },
         "weather_location_result_screen": {
-          "title": "Check Results",
-          "subtitle": "Is your home inside this weather zone?",
-          "send_button": "Send my query!",
-          "back_button": "Try again",
+          "title": "Weather zone to send",
+          "subtitle": "Is your home located in this weather zone?",
+          "send_button": "Yes (send this zone)",
```

Comment:
This is a bit too long for the button. Maybe something like `Yes (send zone)`?
If we can lose 2 characters it'll fit on all screens.

Reply:
resolved in new commit